test/functional/specs/C2581.js

```diff
@@ -66,9 +66,12 @@ const triggerAlloyEvents = ClientFunction(() => {
   return new Promise(resolve => {
     window
       .alloy("event", { viewStart: true, data: { key: "value" } })
-      .then(() => resolve());
-    window.alloy("event", { data: { key: "value" } });
-    window.alloy("event", { data: { key: "value" } });
+      .then(() => {
+        window.alloy("event", { data: { key: "value" } });
+        window.alloy("event", { data: { key: "value" } });
+
+        resolve();
+      });
```

Comment:
It doesn't seem like these tests are testing much at this point. It probably doesn't matter--I refactored these tests in my identity restructure PR because `interact` is getting called for all of the events now: https://github.com/adobe/alloy/pull/392/files#diff-2ab0600556a4e9d90fe1b07dc6c4095dR1-R49